lib/lita/handlers/jira_issues.rb

```diff
@@ -8,15 +8,18 @@ class JiraIssues < Handler
       config :url, required: true, type: String
       config :username, required: true, type: String
       config :password, required: true, type: String
+      config :ignore, default: [], type: Array
 
       route /[a-zA-Z]+-\d+/, :jira_message, help: {
         "KEY-123" => "Replies with information about the given JIRA key"
       }
 
       def jira_message(response)
-        @jira ||= JiraGateway.new(http, config)
-        Set.new(response.matches).each do | key |
-          handle_key(response, key)
+        if !config.ignore.include?(response.user.name)
```

Comment:
I like the [Confident Ruby](http://www.confidentruby.com/) style guards, so something like:

```ruby
def jira_message(response)
  return if config.ignore.include?(response.user.name)
  ...
end
```